src/phoenix/db/enums.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+from typing import Mapping, Type
+
+from sqlalchemy.orm import InstrumentedAttribute
+
+from phoenix.db import models
+
+
+class UserRole(Enum):
+    SYSTEM = "SYSTEM"
+    ADMIN = "ADMIN"
+    MEMBER = "MEMBER"
+
+
+class AuthMethod(Enum):
+    LOCAL = "LOCAL"
+
+
+COLUMN_ENUMS: Mapping[InstrumentedAttribute[str], Type[Enum]] = {
+    models.UserRole.name: UserRole,
+}
```

Comment:
Can we include inside the facilitator?

Reply:
i was actually going to put it directly inside the enum, but too bad it didn't work with older versions of Python. I think this relationship should be colocated with the enum